non-software.md

```diff
@@ -18,6 +18,14 @@ Any open source software license or open license for media (see [above](#data-me
 
 The [SIL Open Font License 1.1](/licenses/ofl-1.1/) keeps fonts open, allowing them to be freely used in other works.
 
+### Hardware
+
+There are the CERN Open Hardware family of licenses: [CERN-OHL-P-2.0](/licenses/cern-ohl-p-2.0/), the permissive variant ([MIT](/licenses/mit/)-like); [CERN-OHL-W-2.0](/licenses/cern-ohl-w-2.0/), the “weakly reciprocal” variant ([MPL](/licenses/mpl-2.0/)-like), and [CERN-OHL-S-2.0](/licenses/cern-ohl-s-2.0/), the “strongly reciprocal” variant ([GPL](/licenses/gpl-3.0/)-like).  The CERN OHL family is also applicable to HDL source code and synthesized bitstreams as would be used in FPGAs.
+
+[CC-BY-4.0](/licenses/cc-by-4.0/) and [CC-BY-SA-4.0](/licenses/cc-by-sa-4.0/) are also commonly used for hardware; however they do not grant patent rights which can restrict usage of patent-encumbered hardware licensed under them and Creative Commons does not recommend them for hardware.
+
+If your hardware has accompanying software then be sure to specify a license for the software portion as well as the hardware portion.
```

Comment:
I would delete this line or fold it into the preceding paragraph like with documentation above.

Reply:
The preceding paragraph does not discuss CC licenses as applying to documentation. It talks about their current use for hardware designs, and explains why that should not be done now that there are better alternatives. Ideally, a hardware project would use one of the CERN licenses for the HW design, one good FOSS license for any software running in the product and one of the CC licenses for documentation (design guide, user guide, etc.). I may have misunderstood your comment. Feel free to use this one as you see fit.

Reply:
Sorry, I meant like in the Documentation section above "If you different licenses..."

Reply:
Though re-reading the Documentation section note addresses something very specific, code included in documentation, so it isn't as analogous as I thought.